src/execution/gas_usage.rs

```diff
@@ -2,16 +2,21 @@ use crate::definitions::constants::*;
 use crate::execution::L2toL1MessageInfo;
 use crate::services::eth_definitions::eth_gas_constants::*;
 
-/// Returns an estimation of the L1 gas amount that will be used (by StarkNet's update state and
-/// the verifier) following the addition of a transaction with the given parameters to a batch;
-/// e.g., a message from L2 to L1 is followed by a storage write operation in StarkNet L1 contract
-/// which requires gas.
-/// Arguments:
-/// l1_handler_payload_size should be an int if and only if we calculate the gas usage of an
-/// InternalInvokeFunction of type L1 handler. Otherwise the payload size is irrelevant, and should
-/// be None.
-/// n_deployments is the number of the contracts deployed by the transaction.
-
+/// Estimates L1 gas usage by Starknet's update state and the verifier
+///
+/// For information about the fee calculation visit the [starknet documentation](https://docs.starknet.io/documentation/architecture_and_concepts/Fees/fee-mechanism/).
+///
+/// # Parameters:
+/// - `l2_to_l1_messages`: A vector of `L2toL1MessageInfo` objects representing the messages from L2 to L1.
```

Comment:
We can link a type's name to its code declaration with [\`type\`] (not sure if it works for parameter names, though):
```rust
/// - [`l2_to_l1_messages`]: A vector of [`L2toL1MessageInfo`] objects representing the messages from L2 to L1.
```